consensus/core/src/core.rs

```diff
@@ -274,8 +281,24 @@ impl Core {
         // only when the validator was supposed to be the leader of the round - so we bring down the missed leaders.
         // Probably proposing for all the intermediate rounds might not make much sense.
 
-        // Consume the ancestors to be included in proposal
-        let ancestors = self.ancestors_to_propose(clock_round, now);
+        // Determine the ancestors to be included in proposal
+        let ancestors = self.ancestors_to_propose(clock_round);
+        self.context
+            .metrics
+            .node_metrics
+            .block_ancestors_count
```

Comment:
Depending on the level of info alternatively could use a histogram?

Reply:
Sounds good.